syne_tune/tuning_status.py

```diff
@@ -36,16 +36,12 @@ def __init__(self):
         self.last_metrics = {}
 
     def add(self, metrics: Dict):
-        for name in metrics.keys():
-            if name not in self.metric_names:
-                self.metric_names.append(name)
-                self.min_metrics[name] = np.inf
-                self.max_metrics[name] = -np.inf
-                self.sum_metrics[name] = 0
-
-        self.min_metrics = {m: min(metrics[m], current) for m, current in self.min_metrics.items()}
-        self.max_metrics = {m: max(metrics[m], current) for m, current in self.max_metrics.items()}
-        self.sum_metrics = {m: metrics[m] + current for m, current in self.sum_metrics.items()}
+        for metric_name, current_metric in metrics.items():
+            if isinstance(current_metric, numbers.Number):
```

Comment:
This should fix the issue indeed, thanks! (I was going to ask about adding this case to the unit-test but you already did it since the first commit :-)

One bug that could happen is that one emits numerical metrics, and then a non numerical one (which I think could happen pretty often). 

Could you either:
* let a TODO comment to deal with this case later
* or store a boolean per metric indicating if they are numerical and throws an error if the type change?